File: tests/fixtures/fixture_java/src/main/java/com/app/models/Order.java
package com.app.models;

public class Order {
    private String id;
    private String userId;
    private double total;

    public Order(String id, String userId, double total) {
        this.id = id;
        this.userId = userId;
        this.total = total;
    }

    public double getTotal() { return total; }

    public static double calculateTotal(double[] items) {
        double sum = 0;
        for (double item : items) sum += item;
        return sum;
    }
}

File: tests/fixtures/fixture_csharp/Models/Order.cs
namespace App.Models
{
    public class Order
    {
        public string Id { get; set; }
        public string UserId { get; set; }
        public decimal Total { get; set; }

        public static decimal CalculateTotal(decimal[] items)
        {
            decimal sum = 0;
            foreach (var item in items) sum += item;
            return sum;
        }
    }
}
